lang/cs/Org.Apache.REEF.Driver/Bridge/DriverBridgeConfigurationOptions.cs

```diff
@@ -137,8 +137,8 @@ public class ContextMessageHandlers : Name<ISet<IObserver<IContextMessage>>>
         {
         }
 
-        [NamedParameter("Whether restart should be enabled on the application", "RestartEnabled", "false")]
-        public class RestartEnabled : Name<bool>
+        [NamedParameter("Evaluator recovery timeout for driver restart. > 0 => restart is enabled.", "DriverRestartEvaluatorRecoveryMillis", "-1")]
```

Comment:
Is milliseconds a convenient unit for this? What would realistic choices look like?